From f0aa149a74c7b0edaa568c2c747d49ad9e1c296e Mon Sep 17 00:00:00 2001
From: tastytea <tastytea@tastytea.de>
Date: Fri, 31 Jan 2020 05:41:23 +0100
Subject: [PATCH] Remove CFLAGS.

---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index d3f54684c..f8b42d003 100644
--- a/Makefile
+++ b/Makefile
@@ -83,8 +83,6 @@ ifdef OPTIMISE
       endif
     endif
   endif
-else
-  CFLAGS += -O
 endif
 
 ifdef DEBUG
-- 
2.24.1

