From 84b4021f3ccb3cfbd12206876e8ec8187c4adea7 Mon Sep 17 00:00:00 2001
From: tastytea <tastytea@tastytea.de>
Date: Thu, 1 Aug 2019 05:44:06 +0200
Subject: [PATCH] Fix branch_name.

---
 mix.exs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mix.exs b/mix.exs
index 399314ef..e78f0b9c 100644
--- a/mix.exs
+++ b/mix.exs
@@ -213,7 +213,7 @@ defmodule Pleroma.Mixfile do
           |> String.trim()
           |> String.replace(identifier_filter, "-")
 
-        "." <> branch_name
+        "-" <> branch_name
       end
 
     build_name =
-- 
2.21.0

